HousingManagementSystemApi/Startup.cs

```diff
@@ -23,6 +24,11 @@ public class Startup
             AssetType.Flat, AssetType.House, AssetType.Dwelling, AssetType.StudioFlat, AssetType.SelfContainedBedsit
         };
 
+        public static readonly IEnumerable<TenureType> EligibleTenureTypes = new List<TenureType>
+        {
+            TenureTypes.Introductory
```

Comment:
This should mirror the list [here](https://github.com/LBHackney-IT/repairs-api-dotnet/blob/90144321c41ca25b87858238a7852a87bc4c0e4c/RepairsApi/V2/Factories/ApiToDomainFactory.cs#L39) to provide consistent behaviour as Repairs Hub.